server/events_test.go

```diff
@@ -2535,6 +2535,84 @@ func TestServerEventsStatszSingleServer(t *testing.T) {
 	checkSubsPending(t, sub, 1)
 }
 
+func TestServerEventsReload(t *testing.T) {
+	conf := createConfFile(t, []byte(`
+		listen: "127.0.0.1:-1"
+		accounts: {
+			$SYS { users [{user: "admin", password: "p1d"}]}
+			test { users [{user: "foo", password: "bar"}]}
+		}
+		ping_interval: "100ms"
+	`))
+	opts := LoadConfig(conf)
+	opts.Trace = true
```

Comment:
Does this test need these to be set to true? Trace, Debug and Verbose?

Reply:
@derek removed